### DISCUSSION TITLE
Just some data on device performance for a Tuya TS0201 (compatible) Temp/Humidity sensor

### DISCUSSION THREAD

**Discussion by adamfred** [Koenkk/zigbee2mqtt#24703] [Devices]
I have two of these devices - connect as TS0201 (compatible I guess)

![image](https://github.com/user-attachments/assets/e0036809-2161-4ac1-a10f-a4bd95008296)

Both work well enough but one burns through its battery regardless of location in the zigbee network. The graph below illustrates two AAA batteries falling below the threshold of 0% battery

![image](https://github.com/user-attachments/assets/35a408ab-adbb-43a7-8531-8e2da4bcf866)

Interview indicates that the sensor that uses too much battery is ZB revision 21 
Sensor without problems reports ZB revision 23

Just reporting this in case it can help people with these sensors.  I will retire the problem sensor, notify the seller and probably buy another pair. At $9AUD they might still be good value.

**Comment by gmeks** [comment]
I have the same issue. Some of them burn the entire batteri in a day or 2
